README.md

```diff
@@ -4,41 +4,57 @@
 
 > This project aims to create toolkit of tools and documentation for generating test data sets based on admin-selected criteria.
 
+> This project a proud part of Salesforce.org Open Source Commons initiative.
+
 **[Read our guide on getting data into your Salesforce Sandbox!](https://sfdo-community-sprints.github.io/DataGenerationToolkit/DataGenGuide)**
 
-### Project Vision/Goals
+### Problem
+Salesforce Admins, Developers, and Consultants need to populate test environments with valid data.  The methods for completing this task are costly, inefficient, or inaccessible.  Utilizing workaround solutions - or worse, making demos/changes in Production - are disruptive and potentially hazardous.
 
-- Everyone has access to an open process/tool to reliably generate sample data for telling user stories.
-- Admin-friendly interface to define requirements for the data generated.
-- Data sets generated through this process could be appopriate for medium-sized orgs and can fit in a sandbox.
-- Data sets generated through this process could be valid for large data valume (LDV) testing.
-- Develop process for how you can build test data sets that follow a story (automated? manual? best practice doc?).
-- Document avialable tools and processes to create and migrate data between orgs.
-- Prioritize leadership development/capacity building through project leadership/contributions from customers, partners, devs, etc.
+### Solution
+We propose a combination of tooling, documentation, and marketing efforts that offer technical and cultural solutions to the problem described above.  By developing tools like Snowfakery, as well as documenting existing tools and methods, we empower Admins, Developers, and Consultants to populate test environments with valid data sets.
```

Comment:
Not sure what "marketing efforts" means in this context...
And by "cultural solutions" do you mean "culturally diverse solutions"?

Reply:
Thank you for the edits!  by marketing, I meant creating content like blogs, documentation, speaking on podcasts, events etc.

By cultural, I meant influencing admins to change their practices to include better testing procedures.  So changing the general culture of how admins do their work.  Cultural diversity is important too but not quite what I was aiming for here!